packages/ui/components/navigation/tabs/HorizontalTabItem.tsx

```diff
@@ -27,10 +27,11 @@ const HorizontalTabItem = function ({
   avatar,
   ...props
 }: HorizontalTabItemProps) {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
   const { t, isLocaleReady } = useLocale();
-  const { asPath } = useRouter();
 
-  const isCurrent = asPath === href;
+  const isCurrent = `${pathname}?${searchParams.toString()}` === href;
```

Comment:
Made this change here [`85cbb37` (#9105)](https://github.com/calcom/cal.com/pull/9105/commits/85cbb373dc265031bd6797fefd787b0d67f99065) cc @zomars 

`asPath` used to have both `pathname` and `href`
This is to get HorizontalTab being shown as selected.

Reply:
What happens of you have an extra search parameter in there? Will it fail the equality?

Reply:
Yeah. So it's not a good implementation. We should check for the presence of the query param and not the entire search string. Even the order of params matter right now.